src/modules/previewpane/powerpreviewTest/powerpreviewTest.vcxproj

```diff
@@ -85,7 +85,7 @@
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
-    <LinkIncremental>false</LinkIncremental>
+    <LinkIncremental>true</LinkIncremental>
```

Comment:
Is this a necessary change ?

Reply:
Yes, this should have been a default value. The build was failing for the Release. https://docs.microsoft.com/en-us/cpp/build/reference/incremental-link-incrementally?view=vs-2019